dtd-parser/pom.xml

```diff
@@ -294,6 +294,15 @@
         </plugins>
     </build>
 
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
use the latest version instead (4.13.2, I think)

Reply:
Fixed in https://github.com/eclipse-ee4j/jaxb-dtd-parser/pull/41/commits/2ac4c77fa9702c42fc217fd0444fd1c1b64f422d